actions/push-to-dockerhub/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: Publish Docker image
+description: Publish docker images to DockerHub
+
+inputs:
+  repository:
+    description: |
+      The caller workflow's repository
+  tags:
+    description: |
+      List of Docker images to be pushed.
+    required: true
+  build_path:
```

Comment:
Sorry, one final nitpick: I think it would be better to name this parameter "context" as that's what it is called Dockerland.